src/libraries/System.Private.Xml/src/System/Xml/Serialization/ReflectionXmlSerializationReader.cs

```diff
@@ -17,17 +18,23 @@ namespace System.Xml.Serialization
 
     internal class ReflectionXmlSerializationReader : XmlSerializationReader
     {
+#pragma warning disable IL2026 // Methods annotated with 'RequiresUnreferencedCodeAttribute' require dynamic access otherwise can break functionality when trimming application code
```

Comment:
This doesn't work for the linker since it can't see `#pragma`s. We will need to suppress them with `UnconditionalSuppressMessage` attributes.

As an aside - we know the types here, isn't there a way to make this safe?

Reply:
(also -while we are in here, can we change these to be `readonly`? I don't think anyone sets them.)